mos-platform/nes-mmc1/prg-rom-banked.ld

```diff
@@ -108,5 +172,23 @@ OUTPUT_FORMAT {
   FULL(prg_rom_13)    FULL(reset, 0, LENGTH(prg_rom_13) ? LENGTH(reset) : 0)
   FULL(prg_rom_14)    FULL(reset, 0, LENGTH(prg_rom_14) ? LENGTH(reset) : 0)
   FULL(prg_rom_fixed) FULL(reset_fixed)
+
+  FULL(prg_rom_16)    FULL(reset, 0, LENGTH(prg_rom_16) ? LENGTH(reset) : 0)
+  FULL(prg_rom_17)    FULL(reset, 0, LENGTH(prg_rom_17) ? LENGTH(reset) : 0)
+  FULL(prg_rom_18)    FULL(reset, 0, LENGTH(prg_rom_18) ? LENGTH(reset) : 0)
+  FULL(prg_rom_19)    FULL(reset, 0, LENGTH(prg_rom_19) ? LENGTH(reset) : 0)
+  FULL(prg_rom_20)    FULL(reset, 0, LENGTH(prg_rom_20) ? LENGTH(reset) : 0)
+  FULL(prg_rom_21)    FULL(reset, 0, LENGTH(prg_rom_21) ? LENGTH(reset) : 0)
+  FULL(prg_rom_22)    FULL(reset, 0, LENGTH(prg_rom_22) ? LENGTH(reset) : 0)
+  FULL(prg_rom_23)    FULL(reset, 0, LENGTH(prg_rom_23) ? LENGTH(reset) : 0)
+  FULL(prg_rom_24)    FULL(reset, 0, LENGTH(prg_rom_24) ? LENGTH(reset) : 0)
+  FULL(prg_rom_25)    FULL(reset, 0, LENGTH(prg_rom_25) ? LENGTH(reset) : 0)
+  FULL(prg_rom_26)    FULL(reset, 0, LENGTH(prg_rom_26) ? LENGTH(reset) : 0)
+  FULL(prg_rom_27)    FULL(reset, 0, LENGTH(prg_rom_27) ? LENGTH(reset) : 0)
+  FULL(prg_rom_28)    FULL(reset, 0, LENGTH(prg_rom_28) ? LENGTH(reset) : 0)
+  FULL(prg_rom_29)    FULL(reset, 0, LENGTH(prg_rom_29) ? LENGTH(reset) : 0)
+  FULL(prg_rom_30)    FULL(reset, 0, LENGTH(prg_rom_30) ? LENGTH(reset) : 0)
+  FULL(prg_rom_fixed) FULL(reset_fixed)
```

Comment:
Like the earlier sections, we should only include this if the size is actually over 512. We can't just check whether `prg_rom_fixed` has a non-zero size though, since it always will. Accordingly, we'll need to use expression like `__prg_rom_size >= 512 ? 0x4000 - 12 : 0` here for both `prg_rom_fixed` and `reset_fixed`.

The syntax is `FULL(<section>, <start>, <length>)` to trim a section's output as it's added to the ROM file. The `<start>` should always be 0, but the `<length>` should either be 0 or the size of the section (`LENGTH(<section>)`), depending on whether or not we have 512 PRG-ROM.